ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/dcat/DcatCatalogValidator.java

```diff
@@ -0,0 +1,13 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.dcat;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.dcat.blanknodevalidators.DcatBlankCatalogNode;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.dcat.cannotcontainvalidators.CannotContainDatasetValidator;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.dcat.cannotcontainvalidators.CannotContainServiceValidator;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DcatCatalogValidator extends DcatValidator {
+	DcatCatalogValidator() {
```

Comment:
suggestion: Unless, there's a good reason to have no access modifier, I'd prefer if you would public access modifier here.